src/main/java/com/strandls/observation/es/util/ConstructESDocument.java

```diff
@@ -39,9 +39,12 @@
 				+ "place_name, reverse_geocoded_name, flag_count, geo_privacy, last_revised,  "
 				+ "visit_count, is_checklist, to_date,  " + "checklist_annotations, "
 				+ "is_locked, language_id, location_scale,  "
-				+ "dataset_id, dataset_title, repr_image_id, repr_image_url, protocol, no_of_images, no_of_videos, no_of_audio, "
+				+ "dataset_id, dataset_title,data_table_id,data_table_title, repr_image_id, repr_image_url, protocol, no_of_images, no_of_videos, no_of_audio, "
```

Comment:
Query is changed, Did you made the same change the main query.? Please update that as well

Reply:
yes, and i have updated the ng-98 ticket as well

Reply:
![Screenshot from 2021-08-12 15-31-38](https://user-images.githubusercontent.com/30981535/129178434-5fcd45f1-f70a-4ccf-94fc-a66825b02f2a.png)